focus-rune.lic

```diff
@@ -0,0 +1,109 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#focus-rune
+=end
+
+custom_require.call(%w[common-travel common common-items common-arcana spellmonitor])
+
+class Runestone
+  include DRC
+  include DRCT
+  include DRCI
+  include DRCA
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'spell',
+          regex: /^[A-z\s\-']+$/i,
+          variable: true,
+          description: "Specify spell to cast from runestone. Spells are: 'Eagle's Cry'
+                'Protection from Evil' Refresh 'Glythtide's Gift' Shadows 'Arc Light'
+                Geyser Bless 'Fire Shards' Althleticism Calm Zephyr Compost
+                'Clear Vision.' Use double quotes." },
+
+        { name: 'debug',
+          regex: /debug/i,
+          optional: true }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+    spell = args.spell
+    @debug = args.debug
+    settings = get_settings
+    @runestone_storage = settings.runestone_storage
+    @harness = settings.runestone_harness
+    runestone_purchase = settings.runestone_purchase
+
+    echo spell if @debug
+    echo @harness if debug
+    @rune = spell_name(spell)
+    echo @rune if @debug
+
+    ### Because Protection from Evil has a lowercase letter for the middle word
+    ### this work around is required.
+    if spell == "protection from evil"
+      temp = "Protection from Evil"
+    else
+      temp = spell.split(/ |\_|\-/).map(&:capitalize).join(" ")
+    end
+    echo temp if @debug
```

Comment:
Consider putting this into its own method, something like:
```ruby
def format_spell_name(spell)
  if spell == "protection from evil"
    return "Protection from Evil"
  else
    return spell.split(/ |\_|\-/).map(&:capitalize).join(" ")
  end
end
```
Then you can call this earlier in the script when you grab the spell argument:
```ruby
spell = format_spell_name(args.spell)
```

Reply:
Added in.